### ISSUE TITLE
GPU memory increase during validation

### ISSUE THREAD

**Issue by siny1998** [MIC-DKFZ/nnUNet#1257]
Hi, authors

Thanks very much for sharing this wonderful code.
I used the nnUNet framework to train our model. I formed "splits_final.pkl" to set the data split for the training set and test set. After training 1000 epochs, the validation for the test set will then start. During the validation process, I found that the GPU memory gradually increases. I conducted the experiment using a 3090 GPU and the version of PyTorch is 1.7.0.  However, when using a 2080Ti GPU, the use of GPU memory is fixed.
Can you give me some useful suggestions?

Best,
Leuan

**Comment by Lars-Kraemer** [comment]
Hi @siny1998, apologies for letting this issue stale for so long.
Are you still encountering this problem with updating pytorch and using the latest version of nnUNet?

Best,
Lars
